.vscode/settings.json

```diff
@@ -3,18 +3,5 @@
   "editor.formatOnSave": true,
   "prettier.requireConfig": true,
   "coverage-gutters.showLineCoverage": true,
-  "cSpell.words": [
-    "Autonumeric",
```

Comment:
Does capitalization matter? some of these are lowercase and some (which I'm assuming are proper nouns) are not.

Reply:
should we make everything lowercase then just as a standard for these lists so theres no confusion in the future if someone wants to add something?

Reply:
I can. The VSCode extension’s quick fix action adds things as they were in that location. Which is how some of the capitalized ones got added.

Reply:
ooooh that will probably be an easier way for ppl to add it in the future than manually typing things in. So i guess its a better practice to just leave it be.

Reply:
I went ahead and updated the current ignores (I hadn’t seen your last message). We don’t have to worry about it moving forward though.